diff --git a/node_modules/react-native-ssh-sftp/android/build.gradle b/node_modules/react-native-ssh-sftp/android/build.gradle
index cd4d6cd..aaf13e0 100644
--- a/node_modules/react-native-ssh-sftp/android/build.gradle
+++ b/node_modules/react-native-ssh-sftp/android/build.gradle
@@ -31,9 +31,9 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.jcraft:jsch:0.1.54'
-    compile 'com.fasterxml.jackson.core:jackson-core:2.9.0'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.0'
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.jcraft:jsch:0.1.54'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.9.0'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.0'
 }
   
\ No newline at end of file
